tests/qos/files/mellanox/qos_param_generator.py

```diff
@@ -171,6 +171,8 @@ def calculate_parameters(self):
         wm_q_shared_lossless = self.qos_params_mlnx[self.speed_cable_len]['wm_q_shared_lossless']
         wm_q_shared_lossless['pkts_num_trig_ingr_drp'] = pkts_num_trig_ingr_drp
         wm_q_shared_lossless['cell_size'] = self.cell_size
+        # It was 8 but recently it failed in rare case. To stabilize the test, increase it to 9
+        wm_q_shared_lossless['pkts_num_margin'] = 9
```

Comment:
@stephenxs , #4661 also has similar changes but diff margin values

Reply:
> @stephenxs , #4661 also has similar changes but diff margin values

Yes, I understand. They are two different issues but will be fixed in similar ways.
Do we need to merge them into one PR? Personally, I think keep both as independent PRs should be ok. How do you think?